### DISCUSSION TITLE
Is it possible to somehow install 32bit version of SEB/SEB configuration tool on 64bit PC in order to generate appropriate BEK key?

### DISCUSSION THREAD

**Discussion by alikhan-ab** [SafeExamBrowser/seb-win-refactoring#738] [Q&A] [answered]
I need to generate 32-bit and 64-bit specific BEK keys. I only have a 64-bit PC. Is it somehow possible to install SEB as a 32bit program on Windows 11 Pro?

**Comment by dbuechel** [comment, marked as answer]
Yes certainly, you can simply install the 32-bit version of SEB (using the respective MSI setup).

**Reply by alikhan-ab** [reply]
Thanks!
